HoloRepositoryDemoApplication/Assets/HoloStorageConnector/Scripts/HoloStorageClient.cs

```diff
@@ -8,12 +8,18 @@
 using System.Reflection;
 using SimpleJSON;
 
-namespace HoloRepository
+
+/// <summary>
+/// Class <c>HoloStorageClient</c> provided multiple methods to retrieve data from Storage server.
+/// </summary>
+namespace HoloStorageConnector
 {
-    public class StorageConnectionServer : MonoBehaviour
+    public class HoloStorageClient : MonoBehaviour
     {
         #region Properties
-        private static string BaseUri = "http://localhost:3001/api/v1";
+        private static string BaseUri = "http://localhost:3001";
```

Comment:
Could you also extract port please? I think it is useful, because for production we will use another port (80). Also the apiPrefix is probably needed for all the code below, so rather than having that separate, could do 
```csharp
private static string BaseUri = $"{StorageAccessorEndpoint}:{Port}/api/v{apiVersion}";
```